src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -38,21 +39,49 @@ public static DeviceTopology createTopology(String zone, String... devices) {
 		return topology;
 	}
 
-	/** Create a radio info entry with the given channel on a given band. */
+	/**
+	 * Create a radio info entry for the given band and channel.
+	 *
+	 * @param band    band
+	 * @param channel channel number
+	 * @return a radio info entry as a {@code JsonObject}
+	 */
+	private static JsonObject createDeviceStatusRadioObject(String band,
+		int channel
+	) {
```

Comment:
formatting: if parameters/arguments don't fit on the first line, none of them should be on the first line; if they don't all fit on the second line, then 1 per line.
```
	private static JsonObject createDeviceStatusRadioObject(
		String band, int channel
	) {
```